### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
situational awareness widget to view data in a card from json

### CODE REVIEW

File: view.controller.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                query: $scope.config.query
+            };
+            var pagedTotalData = new PagedCollection($scope.config.customModule, null, null);
+            pagedTotalData.loadByPost(filters).then(function () {
+                if (pagedTotalData.fieldRows.length === 0) {
+                    $scope.filterValidation = true;
+                    return;
+                }
+                $scope.widgetData = pagedTotalData.fieldRows[0][$scope.config.customModuleField].value;
+                if ('data' in $scope.widgetData) {
```
Comment (cs-pranali-chavan) [inline view.controller.js]: checking as property will be more precise

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
overall looks good

File: view.controller.js (hunk 1 above)
Comment (cs-yash-patodkar) [reply view.controller.js]: done

**Review by cs-pranali-chavan — APPROVED** [review]
Changes are appropriate

**Review by cs-suyog-jinturkar — APPROVED** [review]
Pranali approved these changes already

**Pull request merged**